Added gen_test_dir as arg

nargizas opened the pull request:
Added gen_test_dir as arg in selection_and_ranking.py and made some minor fixes concerning

nargizas committed:
Added gen_test_dir as arg

Made some minor fixes concerning

smkang96 reviewed (changes requested):
Nargiz, sorry for the late response, but could you minimize the diff so that it only contains the `gen_test_dir` change? Right now there is a lot of spurious whitespace changes, which makes the diff difficult to read.

nargizas committed:
Fix gen_test stripping, add gen_tests_dir arg for ranking, TODO for FIB cluster test sampling

The gen_test stripping issue was resolved by checking the presence of backticks and splitting the generated test. The new gen_tests_dir argument was added in selection_and_ranking.py to allow specifying a directory path. The FIB cluster test sampling needs to be fixed.

nargizas committed:
Merge branch 'minor_fixes' of https://github.com/nargizas/libro into minor_fixes

nargizas committed:
removed whitespaces

nargizas committed:
removed unnecessary print()

smkang96 reviewed (approved):
:+1:

smkang96 committed:
Merge branch 'main' into minor_fixes
